src/main/java/hudson/plugins/performance/parsers/LocustParser.java

```diff
@@ -0,0 +1,104 @@
+package hudson.plugins.performance.parsers;
+
+import hudson.plugins.performance.data.HttpSample;
+import hudson.plugins.performance.reports.PerformanceReport;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Date;
+import java.util.List;
+
+public class LocustParser extends AbstractParser
+{
+    enum ReportColumns
+    {
+        Method(0), Name(1), Requests(2), Failures(3), Median(4),
+        Average(5), Min(6), Max(7), AvgContentSize(8), Rps(9);
+
+        int column;
+
+        ReportColumns(final int order)
+        {
+            this.column = order;
+        }
+
+        int getColumn()
+        {
+            return column;
+        }
+    }
+
+    public LocustParser(final String glob, final String percentiles, final String filterRegex)
```

Comment:
Add additional rules to https://github.com/jenkinsci/performance-plugin/blob/master/src/main/java/hudson/plugins/performance/parsers/ParserDetector.java#L32 
for recognize that it's a Locust file format and UT for it